sklearn/metrics/pairwise.py

```diff
@@ -766,8 +766,9 @@ def pairwise_distances_argmin(X, Y, *, axis=1, metric="euclidean", metric_kwargs
 
     See Also
     --------
-    sklearn.metrics.pairwise_distances
-    sklearn.metrics.pairwise_distances_argmin_min
+    pairwise_distances : Distances between pairs of elements of X and Y.
```

Comment:
Let's be a bit more precise to avoid confusion with paired_distances
```suggestion
    pairwise_distances : Distances between every pair of samples of X and Y.
```